concertfit/Controllers/HomeController.cs

```diff
@@ -15,36 +15,41 @@ namespace concertfit.Controllers
     public class HomeController : Controller
     {
         [HttpGet]
-        public List<ConcertfitResponse> Get()
+        public JsonResult Get()
```

Comment:
The group is returning a List to the .cshtml page.  This is likely the functionality they want.  This change assumes they want a JSON stream instead... that might not be the case.